package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yext/search-core",
-  "version": "2.5.2",
+  "version": "2.6.0.beta",
```

Comment:
super nit: can we follow the past version format from Search and use hyphen instead? ([example](https://www.npmjs.com/package/@yext/chat-ui-react?activeTab=versions)L 2.6.0-beta)